eureka2-integration/src/test/java/com/netflix/eureka2/integration/ReadWriteClusterIntegrationTest.java

```diff
@@ -29,13 +31,17 @@
 public class ReadWriteClusterIntegrationTest {
 
     @Rule
-    public final EurekaDeploymentResource eurekaDeploymentResource = new EurekaDeploymentResource(3, 6);
+    public final EurekaDeploymentResource eurekaDeploymentResource = new EurekaDeploymentResource(1, 1);
```

Comment:
Why change this to 1, 1? Part of this cluster wide integration test was to have replication also in play, and this loses that.